.github/workflows/ci.yml

```diff
@@ -0,0 +1,70 @@
+name: cppfront CI
+on: push
+jobs:
+  build-windows:
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ilammy/msvc-dev-cmd@v1
+      - name: Compiler name & version
+        run: cl.exe
+      - name: Build
+        run: cl.exe source/cppfront.cpp -std:c++latest -MD -EHsc -experimental:module -W4
+  build-unix-like:
+    strategy:
+      fail-fast: false
+      matrix:
+        runs-on: [ubuntu-20.04, ubuntu-22.04, macos-latest]
```

Comment:
[Question] What is the added value of running older compilers on ubuntu-20.04 and the newer ones on ubuntu-22.04? Running all the versioned g++ and clang variants on ubuntu-22.04 and skipping ubuntu-20.04 altogether would shorten  the list of exclusions by half, wouldn't it?